apps/web/components/Cookiebar/Cookiebar.vue

```diff
@@ -32,7 +32,7 @@
         </template>
       </div>
     </div>
-    <div v-else class="max-sm:h-full overflow-y-auto h-80 pb-2">
+    <div v-else class="sm:h-1/2 max-sm:h-full overflow-y-auto h-80 pb-2">
```

Comment:
leaving h-full for mobile will result in:

<img width="486" alt="image" src="https://github.com/user-attachments/assets/f5dd3240-213b-416d-a24f-26bebf4b6190">

please test that the scrolling works for further settings as well on mobile.

Reply:
also please check that the desktop version of the cookiebar remains unchanged:

<img width="1254" alt="image" src="https://github.com/user-attachments/assets/d826534e-18d1-4872-ab94-4fe99beb8129">